packages/genesys-spark-tokens/reports/used-undefined-tokens.json

```diff
@@ -1 +1 @@
-[]
+["--gse-ui-button-icon-only-width", "--gse-ui-rte-codeblock-border-color"]
```

Comment:
does this need to be resolved?

Reply:
Think this occured due to the token changes so `--gse-ui-button-icon-only-width` became `--gse-ui-button-iconOnly-width` and  `--gse-ui-rte-codeblock-border-color` the `b` needed to be capitalised `--gse-ui-rte-codeBlock-border-color`